hurumap_land/sql/redistributionprogrammeoutcomebyear.sql

```diff
@@ -0,0 +1,325 @@
+province	GT	2016	2015/2016	benefited households	3
+province	KZN	2016	2015/2016	benefited households	15
+province	LIM	2016	2015/2016	benefited households	14
+province	MP	2016	2015/2016	benefited households	35
+province	NC	2016	2015/2016	benefited households	42
+province	NW	2016	2015/2016	benefited households	20
+province	WC	2016	2015/2016	benefited households	8
+province	EC	2016	2016/2017	benefited households	54
+province	FS	2016	2016/2017	benefited households	2
+province	GT	2016	2016/2017	benefited households	6
+province	KZN	2016	2016/2017	benefited households	116
+province	LIM	2016	2016/2017	benefited households	15
+province	MP	2016	2016/2017	benefited households	170
+province	NC	2016	2016/2017	benefited households	11
+province	NW	2016	2016/2017	benefited households	37
+province	WC	2016	2016/2017	benefited households	85
+province	EC	2016	2017/2018	benefited households	25
+province	FS	2016	2017/2018	benefited households	23
+province	GT	2016	2017/2018	benefited households	2
+province	KZN	2016	2017/2018	benefited households	358
+province	LIM	2016	2017/2018	benefited households	12
+province	MP	2016	2017/2018	benefited households	16
+province	NC	2016	2017/2018	benefited households	10
+province	NW	2016	2017/2018	benefited households	32
+province	WC	2016	2017/2018	benefited households	0
+country	ZA	2016	2009/2010	projects transferred	374
+country	ZA	2016	2010/2011	projects transferred	291
+country	ZA	2016	2011/2012	projects transferred	311
+country	ZA	2016	2012/2013	projects transferred	185
+country	ZA	2016	2013/2014	projects transferred	155
+country	ZA	2016	2014/2015	projects transferred	144
+country	ZA	2016	2015/2016	projects transferred	104
+country	ZA	2016	2016/2017	projects transferred	85
+country	ZA	2016	2017/2018	projects transferred	75
+country	ZA	2016	2009/2010	benefited beneficiaries	11362
+country	ZA	2016	2010/2011	benefited beneficiaries	2697
+country	ZA	2016	2011/2012	benefited beneficiaries	1212
+country	ZA	2016	2012/2013	benefited beneficiaries	430
+country	ZA	2016	2013/2014	benefited beneficiaries	2363
+country	ZA	2016	2014/2015	benefited beneficiaries	370
+country	ZA	2016	2015/2016	benefited beneficiaries	464
+country	ZA	2016	2016/2017	benefited beneficiaries	1271
+country	ZA	2016	2017/2018	benefited beneficiaries	1348
+country	ZA	2016	2009/2010	benefited households	2944
+country	ZA	2016	2010/2011	benefited households	774
+country	ZA	2016	2011/2012	benefited households	354
+country	ZA	2016	2012/2013	benefited households	177
+country	ZA	2016	2013/2014	benefited households	110
+country	ZA	2016	2014/2015	benefited households	280
+country	ZA	2016	2015/2016	benefited households	191
+country	ZA	2016	2016/2017	benefited households	496
+country	ZA	2016	2017/2018	benefited households	478
+\.
+
+ALTER TABLE IF EXISTS ONLY public.redistributionprogrammeoutcomebyyear
+    ADD CONSTRAINT pk_redistributionprogrammeoutcomebyyear PRIMARY KEY (geo_level, geo_code, geo_version, "outcome of redistribution programme");
+
```

Comment:
`year` should be included in `pk_redistributionprogrammeoutcomebyyear`, otherwise there is data duplication:
```
ERROR:  could not create unique index "pk_redistributionprogrammeoutcomebyyear"
DETAIL:  Key (geo_level, geo_code, geo_version, "outcome of redistribution programme")=(province, KZN, 2016, benefited beneficiaries) is duplicated.
```